src/subsplit_dag.cpp

```diff
@@ -338,10 +346,10 @@ EigenVectorXd SubsplitDAG::BuildUniformOnTopologicalSupportPrior() const {
 Node::NodePtrVec SubsplitDAG::GenerateAllTopologies() const {
   std::vector<Node::NodePtrVec> topology_below(NodeCount());
 
-  auto GetSubtopologies = [&topology_below](const SubsplitDAGNode *node) {
+  auto GetSubtopologies = [&topology_below](SubsplitDAGNode node) {
     Node::NodePtrVec rotated_subtopologies, sorted_subtopologies;
     for (const bool rotated : {false, true}) {
-      for (const auto &child_id : node->GetLeafward(rotated)) {
+      for (const auto &child_id : node.GetLeafward(rotated)) {
```

Comment:
We are dropping all `const` refs for iterating over `NeighborView`s as well?

Reply:
Yes, GetLeafward() returns [Const]NeighborsView, which is similar to a string_view: can be cheaply passed by value.